core/modules/search/search.admin.inc

```diff
@@ -137,9 +137,30 @@ function search_admin_settings($form, &$form_state) {
     '#description' => t('Only one type of item will be searched by default. This selection will determine what appears at <a href="@search">@search</a>.', array('@search' => url('search', array('absolute' => TRUE)))),
   );
 
+  $form['pager'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Pager type'),
+  );
+  $pager_type = $config->get('search_pager_type');
+  $form['pager']['search_pager_type'] = array(
+    '#type' => 'radios',
+    '#title' => t('Pager type'),
+    '#title_display' => 'invisible',
+    // The options keys here should be kept the same as the name of theme
+    // function to be called for the pager. See how $variables['pager'] is being
+    // set in template_preprocess_search_results().
+    '#options' => array(
+      'pager' => t('Full pager'),
+      'views_mini_pager' => t('Mini pager'),
```

Comment:
Can we change these keys to "full" and "mini"? Ideally we'd rename `theme_views_mini_pager()` to `theme_mini_pager()` and remove it from being views-specific. That can be a follow-up.

Reply:
Follow-up here: https://github.com/backdrop/backdrop-issues/issues/6096